### ISSUE TITLE
Run baseline_v23 multi-seed training (seed123, seed456)

### ISSUE THREAD

**Issue by wrockey** [wrockey/vmat-diffusion#54]
## Objective
Complete the 3-seed baseline training protocol on v2.3 data (74 cases) to establish seed variability and publishable baseline results.

## Context
- seed42 completed: 200 epochs, 12.78h, best val MAE 6.047 Gy (epoch 172)
- seed42 experienced power outages during training (last-v1.ckpt, last-v2.ckpt evidence)
- Added `--resume` support to training script for power outage recovery

## Checklist
- [x] seed42: training complete (best val MAE 6.047 Gy)
- [x] seed42: inference + evaluation complete (overlap=64)
- [ ] seed123: training
- [ ] seed123: inference + evaluation (overlap=64, gamma_subsample=4)
- [ ] seed456: training
- [ ] seed456: inference + evaluation (overlap=64, gamma_subsample=4)
- [ ] Cross-seed aggregate analysis (mean ± std)
- [ ] Update EXPERIMENTS_INDEX.md with 3-seed results

## Training command
```bash
cd /home/wrockey/projects/vmat-diffusion/scripts
/home/wrockey/miniforge3/envs/vmat-diffusion/bin/python train_baseline_unet.py \
    --data_dir ~/data/processed_npz \
    --exp_name baseline_v23_seed<N> \
    --epochs 200 --batch_size 2 --seed <N> \
    --resume  # auto-recovers from power outages
```

## Power outage mitigation
- `--resume` flag added: auto-detects `last.ckpt` and resumes training
- PyTorch Lightning `save_last=True` already configured
- Training watchdog available: `scripts/training_watchdog.sh`

## Depends on
- #37 (baseline validation) — seed42 portion complete
- #48 (seed42 documentation) — closed

**Comment by wrockey** [comment]
## Progress update

**seed123 training launched** — 2026-02-24
- Command: `python train_baseline_unet.py --data_dir ~/data/processed_npz --exp_name baseline_v23_seed123 --epochs 200 --batch_size 2 --seed 123 --resume`
- `--resume` flag enabled for power outage recovery (#55)
- Estimated completion: ~13 hours (based on seed42's 12.78h)
- Git commit: 95c6069 (includes resume support)

**Comment by wrockey** [comment]
## GPU Queue Update (2026-02-28)

1. ~~3-seed baseline~~ DONE
2. ~~Augmentation ablation (#45)~~ DONE — augmentation helps
3. ~~Combined loss pilot (#57)~~ DONE — PTV gamma 96.4%, D95 overcorrects +1.37 Gy
4. **Architecture scouts (#53) — IN PROGRESS**
   - C11 (AttentionUNet + MSE, seed42) — TRAINING NOW
   - C13 (BottleneckAttn + MSE, seed42) — queued
   - C15 (Wider Baseline + MSE, seed42) — queued
5. Asymmetric weight tuning — parked until scouts done or bigger dataset arrives
6. Full ablation study (#43) — after scouts + weight tuning

**GPU status:** RTX 3090, running C11
